### PR TITLE
Feeder Subsystem Created

### PR DESCRIPTION
**Pull request by David-Stocking**
Created the feeder subsystem and created constants for feeder.

### CODE REVIEW

File: BragiCode/src/main/java/frc/robot/subsystems/Feeder.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+public class Feeder extends SubsystemBase {
+  /** Creates a new Feeder. */
+  private TalonFX FeederMotor1;
+  private TalonFX FeederMotor2;
+  public FeederEnumState mFeederEnumState;
+
+  public enum FeederEnumState{
+    S_WaitingOnIntake, S_DriverReady
+  }
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Feeder.java:21]: How is mFeederEnumState changed?

This looks like everything is public. To change the state, the code would look like:

`feeder.mFeederEnumState = Feeder:S_WaitingOnIntake;`

Usually a setter function would be used:

`feeder.SetState(Feeder:S_WaitingOnIntake); `

You can also have explicit state functions `feeder.WaitOnIntake()` for example.

File: BragiCode/src/main/java/frc/robot/subsystems/Feeder.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Feeder extends SubsystemBase {
+  /** Creates a new Feeder. */
+  private TalonFX FeederMotor1;
+  private TalonFX FeederMotor2;
+  public FeederEnumState mFeederEnumState;
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Feeder.java:17]: mFeederEnumState is not initialized.  What is the starting state for mFeederEnumState?

**Comment by duane701** [comment]
FTP

A few comments to resolve.

File: BragiCode/src/main/java/frc/robot/subsystems/Feeder.java (hunk 1 above)
Comment (David-Stocking) [reply BragiCode/src/main/java/frc/robot/subsystems/Feeder.java:21]: Yes, we are using it publicly

**Comment by David-Stocking** [comment]
PTAL

File: BragiCode/src/main/java/frc/robot/subsystems/Feeder.java (hunk 2 above)
Comment (duane701) [reply BragiCode/src/main/java/frc/robot/subsystems/Feeder.java:17]: Initialized in the constructor

**Comment by duane701** [comment]
LGTM

**Pull request merged**